ext-meta-generator/src/test/resources/output/NestedClassFields.java

```diff
@@ -0,0 +1,30 @@
+package package_name.generated;
+
+import javax.annotation.processing.Generated;
+
+@Generated("tech.ydb.yoj.generator.FieldGeneratorAnnotationProcessor")
+public class NestedClassFields {
+    public static final String FIELD = "field";
+    public class A1 {
```

Comment:
issue: looks like we need a separate field with "a1" for filtering by object:
```
.query(NestedClassFields.A1).eq(obj)
```
It won't work with UpperCase classes like A1 because we need field String A1 for and class A1 here, but I think that there aren't any cases instead of tests

Reply:
I believe we can name them with `_INTEGRAL` (or `OBJ`) suffix, so it will be clear differentiate them. Because if one type something like `EventFields.SOURCE` and `EventFields.Source`, it may confuse them which to use.  And also we will avoid name clashes in case of  names-abbreviations